dns_base/bind/named.conf

```diff
@@ -11,7 +11,8 @@ options {
 	recursion yes;
 
 	auth-nxdomain no;
-	dnssec-validation auto;
+	dnssec-enable no;
+	dnssec-validation no;
```

Comment:
Unfortunately, had to disable this. Somehow the bind servers stopped working.
Our users don't rely on this yet at the moment, but we do need to address it in the future.